packages/devtools-launchpad/src/utils/L10N.js

```diff
@@ -1,9 +1,12 @@
 // @flow
 
 const { sprintf } = require("devtools-modules");
+const { parse } = require("properties-parser");
+
 let strings = {};
 
 function setBundle(bundle: { [key: string]: string }) {
+  bundle = parse(bundle);
```

Comment:
this runtime strategy is borrowed from [l10n.js](https://dxr.mozilla.org/mozilla-central/source/devtools/shared/l10n.js) which uses a similar runtime parser.

It would be nice to do this statically in a webpack loader, but it ultimately does not make a big difference as this is just for the web environment